content/10.diagnostics.md

```diff
@@ -1,33 +1,54 @@
+Molecular testing has also been applied to broader community surveillance by testing wastewater. [@doi:10.1016/j.scitotenv.2020.138764]
+China has approved 11 different nucleic acid and reagent based tests [@url:http://www.nmpa.gov.cn/WS04/CL2056/375802.html].
+
+#### Amplification and Identification Processes
+Given a sample from a patient, tests require sample pre-processing, followed by library preparation, and then sequencing itself [doi:10.1016/j.biotechadv.2020.107537]. 
+Pre-processing is needed to copy the sample in order to create DNA from the RNA sample, then to produce quantities sufficient for a test by amplifying the DNA.
+Library preparation is the process of adding reagents that cut the sequences into fragments appropriate for use in a PCR machine [doi:10.1016/j.biotechadv.2020.107537].
 
 #### RT-PCR
 
+RT-PCR tests are a type of test which uses reverse transcriptase to translate the viral RNA into DNA for pre-processing and uses PCR testing for sequencing.
+There are different reagents used for library preparation which allow identifying one or more target section(s) with PCR.  [@doi:10.1021/acsnano.0c02624]
 The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
 This test uses real-time reverse transcription PCR (RT-PCR) [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the orf1b of the RNA dependent RNA polymerase (RdRP), the Envelope protein gene (E), and the Nucleocapsid protein gene (N).
 In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
 Specifically this tests utilizes two probes against RdRP of which one is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
 Importantly, this assay did not give any false positive results.
 
+
 #### qRT-PCR
 
-Around the same time, researchers in China developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two regions of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029].
+Around the same time as the above RT-PCR tests, researchers in China developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two regions of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029].
 Specifically, this assay was tested on samples coming from two COVID-19 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
 The assay uses the N gene to screen patients, while the orf1b region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
 In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2 is one.
```

Comment:
```suggestion
In this case the test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
```
I know you did add this text but I would also suggest this change to the existing text